### PR TITLE
New Dreams!

### PR DESCRIPTION
**Pull request by babamandaa**
This PR is more of a lore/flavor thing. I know coming from lil miss shitty balance updates herself this is a tiny bit strange, but I wanted to take a crack at rewriting them, and even if these aren't perfect hopefully they're just a little bit more readable.

Daggers:

> ...you dream of energy. Energy reaching the tips of your fingers, energy yearning for the tips of your toes, energy rushing through your veins towards the tip of your knife. You find yourself on the roof of a building, a pouch of gold you didn't earn on your hip and a man with an authoritative voice shouting behind you. Something within you opens, like a valve, allowing the boundless energy to bubble up to your lips - and before you even realize what you've said, you reminisce on how *good* it felt to tell him to fuck off... and how nicely the dagger feels in your palm, primed to slip itself between his coif and his neck as he leers closer.

Swords:

> ...you dream of calm. A priceless commodity in Rockhill, you simply bask in it, meditating at the top of a great mountain. The azure skies stare down at the serene, rolling plains in front of you. The sword in the grass, pristine as the day it was made, shines radiant and guides your hand to it. You trace it along the sky, etching your name into the clouds, a painter at work with their brush - your sword is not just a tool of war, but a tool of creativity and status. And, as you slowly awake, you feel the world open up, and get just a little bit brighter - as long as you have your brush by your side.

Polearms:

> ...you dream of safety. The grip of the staff in your hand never falters, and those at your back never, *ever* doubt you. Throughout your dream, which seems to stretch on and on forever, you lead hundreds of soldiers, or a flock of sheep, or no one at all - but regardless of who's at your back, all you need to feel safe is the staff at your side. The tip of the spear points you, like the north star. No matter what happens, no matter what natural disaster or thief of volf comes your way, you will never compromise, never waver, never rest. Stand tall, soldier.

Maces:

> ...you dream of power. Your body leaves and fails you, and still you hear the creaking and slow snapping of bone and sinew. Something screams. Someone breaks. You bring the club down on it, anyways. Subdue the regret. Dust yourself off - keep going! You'll shirk your brutality in your next life, where you don't make mistakes. Do what you can with this one, while you're still alive. It's the least you can do, killer.

Axes:

> ...you dream of the wild. You bring your axe into a tree, and it bleeds. You bring your axe down into the earth, and it bleeds. You bring your axe down on a shape, and... everything slows down. Your senses sharpen. The deep, dark canopies of the bog suddenly seem... beautiful, in its own way. Like there's a whole life out there, waiting for you. The axe finishes its course, embedding itself in... something that matters less to you, now. Your instincts sharpen as you wake up. Your soul calls for something more.

Bows:

> ...you dream of focus. You are the eye of the eye of the needle, letting your breathing down as you calculate distance. The line between you and your target shortens as you draw back the bowstring. You put a little bit of your soul into every shot you fire, a little bit of love sent with each great shock of grace and brutality in equal measure. The dream ends abruptly as it cracks and sputters, reeling from the shock of catching an arrow in its neck. If you can kill your own dream, you can kill *anything*. Just be sure to place your shots well, hotshot.

### CODE REVIEW

File: code/datums/skills/combat.dm [hunk 1]
```diff
@@ -96,5 +96,5 @@
 /datum/skill/combat/unarmed
 	name = "Fist-fighting"
 	dreams = list(
-		"...your body is bruised, your breathe haggers, but your harden fists and lower-legs refused to give in, going into actions with punches that landed furied blows from their groin to the top that managed to fractured several bones before dealing a decisive knockout that was followed up by a hefty punch that snapped the spine, refocusing your efforts towards another, deciding to use an delayed kick against the unarmored leg, avoiding the neck and above that can't be reached, knocking one by the wall, another on the table, a third against a few objects and a fourth against fifth, moving into a stomp that dislocated and later shattered bone, you stood there, gasping for air, all bruised and beaten up, yet victories..."
+		"..."
 	)
```
Comment (Gyrandola) [inline code/datums/skills/combat.dm:100]: Why did you remove unarmed?

**Comment by Nakalon** [comment]
Please don't make them so verbose. The way they were before when they were smaller was much better. This looks like a good idea in theory but players will just TLDR.

File: code/datums/skills/combat.dm (hunk 1 above)
Comment (babamandaa) [reply code/datums/skills/combat.dm:100]: girlasd

**Comment by Nakalon** [comment]
I'll ask you again to cut the text amount.

**Comment by Nakalon** [comment]
> This is honestly so much fucking better than the ones mori wrote oh my god

It wasnt Mori it was Redth

**Comment by Nakalon** [comment]
Pretty please, I once again request that you reduce the wall of text to 3 times its size and I will merge.

**Comment by Gyrandola** [comment]
Closing this, reopen it when you have addressed @Nakalon 's requests.
